Resources/doc/content-negotiation.md

```diff
@@ -25,30 +25,45 @@ dunglas_api:
 
 ## Registering a custom format in the Negotiation library
 
-If the format you want to use is not supported by default in the Negotiation library, you must register it in your bundle
-extension:
+If the format you want to use is not supported by default in the Negotiation library, you must register it using a [compiler pass](http://symfony.com/doc/current/components/dependency_injection/compilation.html#creating-a-compiler-pass):
 
 ```php
 // src/AppBundle/DependencyInjection/AppBundleExtension.php
 
-namespace AppBundle\DependencyInjection;
+namespace AppBundle\DependencyInjection\Compiler;
 
 use Symfony\Component\DependencyInjection\ContainerBuilder;
-use Symfony\Component\HttpKernel\DependencyInjection\Extension;
-/**
- * The extension of this bundle.
- *
- * @author Kévin Dunglas <dunglas@gmail.com>
- */
-class AppBundleExtension extends Extension
+use Symfony\Component\DependencyInjection\Compiler\CompilerPassInterface;
+
+class AppBundlePass implements CompilerPassInterface
```

Comment:
Why not. 
But, indirectly, my point about the `AppBundlePass` (It would even been `AppPass`, though) is that the same as for bundles, you do not especially need to create multiple compiler passes for tweaking existing services and behaviors related to your application only (not something reusable).